src/docs/release/breaking-changes/scaffold-messenger.md

```diff
@@ -95,6 +96,22 @@ Typically, the ScaffoldMessenger widget is introduced by the MaterialApp
 at the top of your application widget tree.
 ```
 
+The `ScaffoldMessenger` will also assert if attempting to show `SnackBar`s to nested `Scaffold`s. If
```

Comment:
=> The `ScaffoldMessenger` throws an assertion when attempting to share a `Snackbar` to nested `Scaffold`s.

Question: What does it mean to "show a SnackBar to nested Scaffolds?

Reply:
The ScaffoldMessenger will distribute a SnackBar to all registered Scaffolds. Scaffolds register with the nearest instance of the ScaffoldMessenger to receive SnackBars. If Scaffolds are nested, one inside the other, and they both register with the same ScaffoldMessenger, then that would mean that both would receive a SnackBar when ScaffoldMessenger.showSnackBar(...) is called. If this were allowed (instead of asserting), then two SnackBars would show up in the UI. There is a perfect example of this with pics in: https://github.com/flutter/gallery/issues/376

Reply:
I've updated the error message, let me know if there is a way I can make this clearer. :)